runtime/autoload/dist/ft.vim

```diff
@@ -847,6 +847,27 @@ func dist#ft#FTperl()
   return 0
 endfunc
 
+" LambdaProlog and Standard ML signature files
+func dist#ft#FTsig()
+  if exists("g:filetype_sig")
+    exe "setf " .. g:filetype_sig
+    return
+  endif
+
+  let lprolog_comment = '^\s*\%(/\*\|%\)'
```

Comment:
We try to use `findany` wherever possible, I'll send you an updated version in a few minutes :)